apps/docs/src/app/pages/component-pages/snackbar-page/snackbar-page.component.html

```diff
@@ -0,0 +1,12 @@
+<h1>Snackbar</h1>
+
+<h2>Examples</h2>
+
+<div class="controls">
+  Controls:
+  <button anglifyButton appearance="text" (click)="openSnackbar()">Show</button>
+</div>
```

Comment:
Please add at least one `app-code-example` so that users can see the necessary code right in the browser.

Reply:
Will do.